kie-server-parent/kie-server-controller/kie-server-controller-standalone-war/src/main/shared-ee6-ee7-resources/META-INF/services/org.kie.server.controller.api.storage.KieServerTemplateStorage

```diff
@@ -0,0 +1 @@
+org.kie.server.controller.impl.storage.FileBasedKieServerTemplateStorage
```

Comment:
I belive this will not be included in web container assembly, could you please double check after the build?